reference/6/Microsoft.PowerShell.Utility/Send-MailMessage.md

```diff
@@ -0,0 +1,377 @@
+
+```powershell
+Send-MailMessage -To "User01 <user01@example.com>" -From "ITGroup <itdept@example.com>" -Cc "User02 <user02@example.com>" -bcc "ITMgr <itmgr@example.com>" -Subject "Don't forget today's meeting!" -Credential domain01\admin01 -UseSsl
+```
+
+This command sends an email message from User01 to the ITGroup mailing list with a copy (Cc) to User02 and a blind carbon copy (Bcc) to the IT manager (ITMgr).
+
+The command uses the credentials of a domain administrator and the *UseSsl* parameter.
+
+## PARAMETERS
+
+### -Attachments
+
+Specifies the path and file names of files to be attached to the email message.
+You can use this parameter or pipe the paths and file names to **Send-MailMessage**.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases: PsPath
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Bcc
+
+Specifies the email addresses that receive a copy of the mail but are not listed as recipients of the message.
+Enter names (optional) and the email address, such as `Name <someone@example.com>`.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Body
+
+Specifies the body of the email message.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 2
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -BodyAsHtml
+
+Indicates that the value of the *Body* parameter contains HTML.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: BAH
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Cc
+
+Specifies the email addresses to which a carbon copy (CC) of the email message is sent.
+Enter names (optional) and the email address, such as `Name <someone@example.com>`.
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Credential
+
+Specifies a user account that has permission to perform this action.
+The default is the current user.
+
+Type a user name, such as User01 or Domain01\User01.
+Or, enter a **PSCredential** object, such as one from the Get-Credential cmdlet.
+
+```yaml
+Type: PSCredential
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DeliveryNotificationOption
+
+Specifies the delivery notification options for the email message.
+You can specify multiple values.
+None is the default value.
+The alias for this parameter is **dno**.
+
+The delivery notifications are sent in an email message to the address specified in the value of the *From* parameter.
```

Comment:
Changed as by comment from Chaitu_a in https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/Send-MailMessage?view=powershell-5.1